src/main/java/seedu/duke/todo/DoneCommand.java

```diff
@@ -0,0 +1,19 @@
+package seedu.duke.todo;
+
+public class DoneCommand {
+    private int pageNumber;
```

Comment:
Likewise for this and other commands. Ideally all of our commands should be defined under the "commands" package